content/releases/q/release-1.6.0.md

```diff
@@ -0,0 +1,11 @@
+---
+url: "/releases/q-1.5.0/"
```

Comment:
Is this url correct, should it not be 1.6.0 ?

Reply:
Thanks, fixed.